src/application/services/FileCacheService.ts

```diff
@@ -0,0 +1,65 @@
+import fs from "fs";
+import path from "path";
+
+export const DIG_FOLDER_PATH = path.join(process.cwd(), ".dig");
```

Comment:
Typically this would be in the user directory of the os.

We will need a central way to manage the .dig user directory, as this was something I didn't plan out well in v1 and it caused mixups throughout the code

Reply:
Ok. I've added loading from environment [here ](https://github.com/DIG-Network/dig-sdk-v2/pull/5) and I think i can make a wrapper over that for anything that is path related so we only define the path in relation to the base dig file in the user directory.

Maybe we could define a generic folder structure for the .dig so I can expect what files would be populated there?